### ISSUE TITLE
Unable to compute metrics for model output/segment-10061305430875486848_1080_000_1100_000_with_camera_labels : Error(s) in loading state_dict for LinLayers

### ISSUE THREAD

**Issue by LuoShuaiying023** [yifanlu0227/ChatSim#57]
Scene: output/segment-10061305430875486848_1080_000_1100_000_with_camera_labels
Method: ours_30000
Metric evaluation progress: 0it [00:00, ?it/s]
  SSIM :          nan
  PSNR :          nan
  LPIPS:          nan

Method: ours_30000
Metric evaluation progress:   0%|                                                                                | 0/120 [00:01<?, ?it/s]
Unable to compute metrics for model output/segment-10061305430875486848_1080_000_1100_000_with_camera_labels : Error(s) in loading state_dict for LinLayers:
	Missing key(s) in state_dict: "0.1.weight", "1.1.weight", "2.1.weight", "3.1.weight", "4.1.weight". 
	Unexpected key(s) in state_dict: "features.0.weight", "features.0.bias", "features.2.weight", "features.2.bias", "features.5.weight", "features.5.bias", "features.7.weight", "features.7.bias", "features.10.weight", "features.10.bias", "features.12.weight", "features.12.bias", "features.14.weight", "features.14.bias", "features.17.weight", "features.17.bias", "features.19.weight", "features.19.bias", "features.21.weight", "features.21.bias", "features.24.weight", "features.24.bias", "features.26.weight", "features.26.bias", "features.28.weight", "features.28.bias", "classifier.0.weight", "classifier.0.bias", "classifier.3.weight", "classifier.3.bias", "classifier.6.weight", "classifier.6.bias". 
Oh!!!!!!!!What can I do?

**Comment by tkc123-gs** [comment]
how do you do
